src/main/webapp/app/shared/delete-dialog/delete-dialog.component.html

```diff
@@ -44,7 +44,13 @@ <h4 class="modal-title" [ngSwitch]="actionType">
             [ngSwitch]="actionType"
         >
             <span *ngIf="submitDisabled"><fa-icon [icon]="faSpinner" [spin]="true">&nbsp;</fa-icon>&nbsp;</span>
-            <fa-icon [icon]="useFaCheckIcon ? faCheck : faTimes"></fa-icon>&nbsp;
+            <fa-icon *ngSwitchCase="actionTypes.Delete" [icon]="useFaCheckIcon ? faCheck : faTrash"></fa-icon>&nbsp;
+            <fa-icon *ngSwitchCase="actionTypes.Reset" [icon]="useFaCheckIcon ? faCheck : faUndo"></fa-icon>
+            <fa-icon *ngSwitchCase="actionTypes.Cleanup" [icon]="useFaCheckIcon ? faCheck : faTimes"></fa-icon>
```

Comment:
I do not understand this change - why are further options added here?

Arent Delete, Rest and Cleanup displayed twice with the spans below?

Reply:
The spans below provide different text for the different action types, whereas this changes adds different icons for different actiontypes.

Reply:
Ah I see, I think its a slightly better approach to have one switch case and combine the icon and text within one element (e.g. wrapped with `<></>`), but for this small change I do not have a strong opinion